### PR TITLE
Wi-fi Plugin: Finish Exporting states

### PR DESCRIPTION
**Pull request by DanielCranham**
following morfents commit but updating it for the wi-fi plugin aswell, to avoid issues after hotpatching

### CODE REVIEW

**Comment by Zarel** [comment]
Build error. Don't pullreq without running `npm test` first.

This is your last warning.

File: chat-plugins/wifi.js [hunk 1]
```diff
@@ -29,7 +28,7 @@ function clearGiveaway() {
 	if (wifiRoom) delete wifiRoom.plugin;
 }
 
-var QuestionGiveAway = (function () {
+var QuestionGiveAway = (function	() {
```
Comment (ascriptmaster) [inline chat-plugins/wifi.js]: A ***tab*** after `function`? Are you serious? They're supposed to be single spaces like they used to be!

Comment (DanielCranham) [inline chat-plugins/wifi.js]: Okay I need to sleep after this, I'll fix it up, thanks for pointing it out

File: chat-plugins/wifi.js [hunk 2]
```diff
@@ -82,26 +81,26 @@ var QuestionGiveAway = (function () {
...
+				this.question = value;
+				output.sendReply("The question has been changed to " + value + ".");
+				break;
+			case 'answer':
+				var ans = this.sanitizeAnswers(value);
+				if (Object.keys(ans).length <= 0) return output.sendReply("You must specify at least one answer and must not contain any special characters.");
+
+				this.answers = ans;
+				output.sendReply("The answers have been changed to " + value + ".");
+				break;
```
Comment (ascriptmaster) [inline chat-plugins/wifi.js]: Remove those added indents you put in there. `case` keywords should be in line with the `switch` statement.

**Comment by ascriptmaster** [comment]
You had one error (accidentally used `giveaways` once instead of `giveaway`), so none of these other style changes were even remotely necessary, and to make it worse, they're mostly wrong!

**Comment by DanielCranham** [comment]
Alright, I made some very stupid errors. This should be fixed and ready now.

File: chat-plugins/wifi.js [hunk 3]
```diff
@@ -437,6 +437,17 @@ var commands = {
 	}
 };
 
+var giveawayRoom = Rooms.get('wifi');
+if (giveawayRoom) {
+	if (giveawayRoom.plugin) {
+		giveaway = giveawayRoom.plugin.giveaways;
```
Comment (Zarel) [inline chat-plugins/wifi.js]: Why is this all the way down here now?

Comment (DanielCranham) [inline chat-plugins/wifi.js]: Hmm, I just literally followed a similar style to the jeopardy plugin and wrote it just before exports.commands in the file

Comment (Zarel) [inline chat-plugins/wifi.js]: This isn't good style, but honestly it doesn't make a difference in reality.

**Comment by codelegend** [comment]
Wait, why is the data imported/exported again??
Line 22 imports the data on hotpatching, and ``spawnGiveaway(...)`` takes care of exporting (ie. attaching the object to the wifi room, line 269)
This addition is totally unnecessary.

**Pull request merged**